Documentation/api.md

```diff
@@ -91,6 +91,7 @@ Endpoint defines a scrapeable endpoint serving Prometheus metrics.
 | targetPort | Name or number of the target port of the endpoint. Mutually exclusive with port. | intstr.IntOrString | false |
 | path | HTTP path to scrape for metrics. | string | false |
 | scheme | HTTP scheme to use for scraping. | string | false |
+| params | Optional HTTP URL parameters | map[string]\[]string | false |
```

Comment:
Unfortunately the doc generation tests failed here because they expected `map[string][]string` instead of `map[string]\[]string`, but I'm guessing you tested this out which is why the `\` is in there. If that's the case, we'll need to adapt our code for doc generation.

Reply:
@brancz I didn't realize that the docs were generated from the types. I had thought a backslash was necessary due to markdown reference links. Turns out that's not the case and it's ok without the backslash. 

Now this should be good to go.